cfme/scripting/quickstart.py

```diff
@@ -78,31 +78,60 @@
      " freetype-devel")
 ]
 
+DEBIAN_RELEASE_FILE = '/etc/os-release'
+DEBIAN_PACKAGES_SPEC = [
+    # Extend this
+    ("Ubuntu", "openssl",
+     " python-virtualenv gcc postgresql libxml2-dev"
+     " libxslt1-dev libzmq3-dev libcurl4-openssl-dev"
+     " g++ openssl libffi-dev python-dev libtesseract3"
+     " libpng-dev libfreetype6-dev libssl-dev")
+]
 
 if os.path.exists(REDHAT_RELEASE_FILE):
 
     with open(REDHAT_RELEASE_FILE) as fp:
         release_string = fp.read()
     for release, curl_ssl, packages in REDHAT_PACKAGES_SPECS:
         if release_string.startswith(release):
-            REDHAT_PACKAGES = packages
+            REQUIRED_PACKAGES = packages
             os.environ['PYCURL_SSL_LIBRARY'] = curl_ssl
             break
     else:
         print("{} not known".format(release_string))
         sys.exit(1)
-    assert REDHAT_PACKAGES
+    assert REQUIRED_PACKAGES
 
     if HAS_DNF:
         INSTALL_COMMAND = 'dnf install -y'
     else:
         INSTALL_COMMAND = 'yum install -y'
     if not IS_ROOT:
         INSTALL_COMMAND = 'sudo ' + INSTALL_COMMAND
+
+elif os.path.exists(DEBIAN_RELEASE_FILE):
```

Comment:
REQUIRED: @ohiliazo, you're right. it makes sense to combine both if statements and use os-release everywhere instead.